package.json

```diff
@@ -62,7 +62,12 @@
         "onLanguage:ssm-yaml",
         "onCommand:aws.ssmDocument.createLocalDocument",
         "onCommand:aws.ssmDocument.openLocalDocument",
+        "onCommand:aws.ssmDocument.openLocalDocumentJson",
+        "onCommand:aws.ssmDocument.openLocalDocumentYaml",
```

Comment:
Looks like you deleted a reference further down to `openLocalDocument` now that you have the language-specific ones. Is there a reason to leave this around? Goes for other occurrences within the package.json file (I see one on line 495, for instance)

Reply:
openLocalDocument command is still used for quick download button shown below
<img width="287" alt="Screen Shot 2020-09-21 at 11 36 19 AM" src="https://user-images.githubusercontent.com/70638142/94039067-164ae300-fd7c-11ea-8d5b-7391d268a40b.png">

Reply:
Download button in the explorer node item is invoking "openLocalDocument". I will do a small change here so this action won't prompt the user for the format, rather it will use the format the document was saved in.

There are also 2 context menu actions when you right click on the explorer node item which are using "openLocalDocumentJson" and "openLocalDocumentYaml" commands.